docs/getting-started/install-configure.md

```diff
@@ -460,23 +460,9 @@ Slack][Slack] and our community will highlight it here!</i>
   and Azure mentionded in this guide.
 
 <!-- Named Links -->
-
-[package]: {{<ref "../concepts/packages" >}}
-[provision infrastructure]: {{<ref "provision-infrastructure" >}}
-[create a configuration]: {{<ref "create-configuration" >}}
 [Install]: {{<ref "../reference/install" >}}
```

Comment:
@plumbis it looks like we are now going to be in a place where we have mixed named and inline links. Can you share why `[package]` does not work, but something like `[Install]` does?

Reply:
This was a new one to me. The problem is a result of how Hugo renders shortcodes. The broken links are inside a `tabs` shortcode.

Hugo renders shortcodes from the inside to the outside. The contents of the shortcode are fully rendered and the result is passed back to the original page. 

With named links, the link inside a shortcode doesn't exist. Only the text `[package]`. The Markdown engine has no idea that`[package]` is actually a link. The named link context only exists on the parent page. 

The solution is to not use named links in shortcodes and put the link inline, like the PR does. I would vote to discourage named links across the docs not only because of this problem but it can also create unintended errors if a link name is accidentally reused.

Reply:
Thanks for the explanation @plumbis! I am fine with either solution, but would advocate for consistency as well 👍🏻 I don't think we need to block this important update on that though.